debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-settings (4.2.1-0) unstable; urgency=low
+
+  * Fixed wireless connection on boot for Kano 4.2.0 release
```

Comment:
I'd rewrite this as `Fixed race condition at boot preventing Wi-Fi from reconnecting to existing network`